content/en/tracing/trace_pipeline/trace_retention.md

```diff
@@ -103,6 +100,24 @@ For example, you can create filters to keep all traces for:
 - High-priority customers using a mission-critical feature of your SaaS solution.
 - Specific versions of an online delivery service application.
 
+## Trace search and analytics on indexed spans
+
+### In the Trace Explorer
+
+By default, spans indexed by custom retention filters **and** the intelligent retention filter are included in the Trace Explorer [aggregated views][6] (timeseries, toplist, table)
+
+However, because the diversity sampled set of data which is **not uniformly sampled** (i.e. not proportionnally representative of the full traffic) and biased towards errors and high latency traces, you can choose to exclude these from these views by adding `-retained_by:diversity_sampling` query parameter to the query.
```

Comment:
```suggestion
However, because the diversity-sampled set of data is **not uniformly sampled** (that is, not proportionally representative of the full traffic) and is biased towards errors and high latency traces, you can choose to exclude these spans from these views by adding `-retained_by:diversity_sampling` query parameter to the query.
```